dv_utils/client.py

```diff
@@ -16,18 +17,71 @@
 
 class Client:
     """
-    Http client to interact with the Data-village API.
+    Http client to interact with the Datavillage API.
     """
+    DATA_PROVIDER_COLLABORATOR_ROLE_VALUE="DataProvider"
+
+    DATA_PROVIDER_COLLABORATOR_ROLE_VALUE="DataProvider"
+    DATA_CONSUMER_COLLABORATOR_ROLE_VALUE="DataConsumer"
```

Comment:
why is `DATA_PROVIDER_COLLABORATOR_ROLE_VALUE` declared twice? Should one be `CODE_PROVIDER_*`?

Reply:
✅